modules/landing-page/landing-dashboard-overview.tsx

```
@@ -0,0 +1,346 @@
+              <Button size="sm" variant="ghost" className="h-5 w-5 p-0">
+                <Plus className="h-3 w-3" />
+              </Button>
+            </div>
+            <div className="space-y-1">
+              {projects.map((project, index) => (
+                <a
+                  key={index}
+                  href="#"
+                  className="group flex items-center space-x-3 rounded-md px-3 py-2 text-sm text-gray-600 transition-colors hover:bg-white hover:text-black"
+                >
+                  <div className={`h-2 w-2 rounded-full ${project.color}`}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)